internal/core/accessibility.rs

```diff
@@ -24,6 +30,56 @@ pub enum AccessibleStringProperty {
     ValueStep,
 }
 
+// Defines an accessibility action.
+pub enum AccessibilityAction {
+    Default,
+    Focus,
+    Blur,
+    Collapse,
+    Expand,
+    CustomAction,
+    Decrement,
+    Increment,
+    ReplaceSelectedText(SharedString),
+    ScrollBackward,
+    ScrollDown,
+    ScrollForward,
+    ScrollLeft,
+    ScrollRight,
+    ScrollUp,
+    ScrollIntoView,
+    ScrollToPoint(LogicalPoint),
+    SetScrollOffset(LogicalVector),
+    SetTextSelection(Option<core::ops::Range<i32>>),
```

Comment:
Are those byte offsets or char offsets?

Reply:
I checked how it is handled in `Flickable` and `TextInput` and used it here.

Reply:
I think it is byte offset in `TextInput` right? So it is handled by the renderer.

Reply:
👍